src/v2/Apps/Auction2/Queries/useRefreshUserData.ts

```diff
@@ -0,0 +1,27 @@
+import { Environment, fetchQuery, graphql } from "relay-runtime"
+import { useSystemContext } from "v2/System"
+
+/**
+ * After certain operations (like adding a credit card during a bid), we need
+ * to refresh the user data so that if a bid is below reserve, we aren't
+ * resending a query to add another credit card.
+ */
+export const useRefreshUserData = () => {
+  const { relayEnvironment } = useSystemContext()
+
+  const refreshUserData = () => {
+    return fetchQuery(
+      relayEnvironment as Environment,
+      graphql`
+        query useRefreshUserDataQuery {
+          me {
+            ...Auction2BidRoute_me
```

Comment:
In both mutations as well as queries, this works great for refreshing the relay store / UI -- just pass the fragment in that you want to refresh, and it will update.